src/NzbDrone.Core/Notifications/Ntfy/NtfyProxy.cs

```diff
@@ -114,9 +114,14 @@ private void SendNotification(string title, string message, HttpRequestBuilder r
                     requestBuilder.Headers.Add("X-Click", settings.ClickUrl);
                 }
 
+                if (!settings.AccessToken.IsNullOrWhiteSpace())
```

Comment:
Using `else if` instead of checking again below puts all the authentication related things in one place

```suggestion
                if (!settings.AccessToken.IsNullOrWhiteSpace())
                {
                    requestBuilder.Headers.Add("Authorization", "Bearer " + settings.AccessToken);
                }
                else if (!settings.UserName.IsNullOrWhiteSpace() && !settings.Password.IsNullOrWhiteSpace())
                {
                    request.Credentials = new BasicNetworkCredential(settings.UserName, settings.Password);
                }
```

Reply:
Makes sense!